src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -101,7 +100,6 @@ export function InternalDashboardTopNav({
     dashboardApi.hasRunMigrations$,
     dashboardApi.hasUnsavedChanges$,
     dashboardApi.savedObjectId,
-    dashboardApi.managed$,
```

Comment:
Neat detail here that `managed` can't change anymore during the lifetime of the Dashboard API. I figure this is because the whole API is recreated every time the saved object ID changes. Good catch to find this because the fewer elements that can change the better IMO!